socorro/lib/requestslib.py

```diff
@@ -35,18 +35,27 @@ def send(self, *args, **kwargs):
 
 def session_with_retries(
     total_retries=5,
-    backoff_factor=0.2,
+    backoff_factor=0.1,
```

Comment:
Earlier, I thought the backoff was linear so it'd go 0.2, 0.4, 0.6, 0.8, 1.0. That's wrong--it's exponential. So with a 0.2 backoff factor, it goes 0.2, 0.4, 0.8, 1.6, 3.2.

This changes it to 0.1 which goes 0.1, 0.2, 0.4, 0.8, 1.6.